pkg/resource/transform_job/custom_set_output.go

```diff
@@ -0,0 +1,94 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// Use this file if the Status/Spec of the CR needs to be modified after
+// create/describe/update operation
+
+package transform_job
+
+import (
+	"context"
+
+	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
+	svcapitypes "github.com/aws-controllers-k8s/sagemaker-controller/apis/v1alpha1"
+	"github.com/aws/aws-sdk-go/aws"
+	svcsdk "github.com/aws/aws-sdk-go/service/sagemaker"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// customCreateTransformJobSetOutput sets the resource in TempOutofSync if TransformJob is
+// in creating state. At this stage we know call to createEndpoint was successful.
```

Comment:
nit: `s/createEndpoint/createTransformJob/g`

Reply:
Yea, I fixed these in my next commit, just have not pushed it yet. Thanks though!